src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java

```diff
@@ -164,7 +173,7 @@ public interface InvoiceUserApi extends KillbillApi {
      * @throws InvoiceApiException
      */
     @RequiresPermissions(INVOICE_CAN_TRIGGER_INVOICE)
-    public Invoice triggerInvoiceGeneration(UUID accountId, LocalDate targetDate, CallContext context) throws InvoiceApiException;
+    public Collection<Invoice> triggerInvoiceGeneration(UUID accountId, LocalDate targetDate, CallContext context) throws InvoiceApiException;
```

Comment:
> seems like we are not consistent

Indeed... FWIW, I always go for lower specificity, whenever possible (which is probably why there are so many Iterables everywhere 😁).

> Should we introduce new apis such as `triggerInvoiceGroupGeneration` that would return an `InvoiceGroup`?

What would `triggerInvoiceGeneration` return then, in case a plugin creates a group?

Reply:
As discussed, no change with `triggerInvoiceGeneration` (it wouldn't invoke the new grouping API).

Caller must call `triggerInvoiceGroupGeneration` explicitly to generate the groups.

👍